independent-projects/arc/processor/src/main/java/io/quarkus/arc/processor/ObserverInfo.java

```diff
@@ -91,17 +91,6 @@ static ObserverInfo create(String id, BeanDeployment beanDeployment, DotName bea
             isAsync = context.isAsync();
         }
 
-        if (!transactionPhase.equals(TransactionPhase.IN_PROGRESS) && !jtaCapabilities) {
```

Comment:
The decision from Zulip chat was to tweak the message to:
* not show up in case there is JTA present but no transaction is in progress
* in case there is no JTA it should still log a warning but something a little more verbose such as
  * `The observer %s makes use of %s transactional observers but no JTA capabilities were detected. Transactional observers will be notified at the same time as other observers.`

Reply:
Reverted, and the message is adjusted. Issue is created.